internal/tiltfile/docker.go

```diff
@@ -350,25 +346,15 @@ func (b *customBuild) Hash() (uint32, error) {
 	return 0, fmt.Errorf("unhashable type: custom_build")
 }
 
-const (
-	addFastBuildN = "add_fast_build"
-)
-
 func (b *customBuild) Attr(name string) (starlark.Value, error) {
 	switch name {
-	case addFastBuildN:
-		return starlark.NewBuiltin(name, b.addFastBuild), nil
 	default:
 		return nil, nil
 	}
 }
 
 func (b *customBuild) AttrNames() []string {
-	return []string{addFastBuildN}
-}
-
-func (b *customBuild) addFastBuild(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
-	return &fastBuild{s: b.s, img: b.img}, nil
+	return []string{}
```

Comment:
should leave this method in and have it throw the deprecation error

Reply:
currently you get:
>        Error: custom_build has no .add_fast_build field or method

Reply:
How's this? It will error when you ask for "add_fast_build".